pyalgotrading/algobulls/connection.py

```diff
@@ -462,10 +478,23 @@ def start_job(self, strategy=None, start=None, end=None, instruments=None, lots=
                 'paramValue': parameters[parameter_name]
             })
 
+        # get exchange location
+        exchange = 'NSE'
+        location = SEGMENT_COUNTRY_MAP[exchange]
+
+        if len(instruments[0].split(':')) == 2:
```

Comment:
bring out `instruments[0].split(':')` and store in a variable. For temp variables `_` is the best name. Use `_` at both 485 and 486 lines.

Reply:
Done : renamed to `_exch` since `_` is already used below